src/general/world_effects/UnderwaterVentEruptionEffect.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Newtonsoft.Json;
+using Xoshiro.PRNG64;
+
+/// <summary>
+///   An effect reducing the glucose amount (for the microbe stage to make early game easier, and the late game harder)
```

Comment:
This seems copy-pasted, so could you double check you don't have accidentally unchanged copy-pasted stuff in your PR?